pymatgen/analysis/transition_state.py

```diff
@@ -303,3 +305,92 @@ def as_dict(self):
                 'energies': jsanitize(self.energies),
                 'forces': jsanitize(self.forces),
                 'structures': [s.as_dict() for s in self.structures]}
+
+def combine_neb_plots(root_dirs, arranged_dirs=False, reverse_plot=False):
```

Comment:
This method should not take in a list of rootdirs. It should simply take in a list of NEBAnalysis. Why assume that people are doing the analysis from a set of files?